### DISCUSSION TITLE
2024-05-27(월)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#46] [데일리스크럼]
아래 양식 복사해서 댓글로 작성해주시면 됩니다😃

```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한일
- BottomModal 공통 컴포넌트 구현

### 해야할 일
- BottomModal 코드 리뷰 반영
- 카카오 OAuth 로직 구현

### 어려운 일
- 카카오 OAuth 공부를 틈틈히 하고 있습니다!

**Comment by devwqc** [comment]
### 완료한일
- css colors 변수, z-index 변수, shadow mixin 생성
- SUIT 폰트 적용
- NavTop, NavBottom 구현

### 해야할 일
- Layout 작업

### 어려운 일
- 파이팅!!

**Comment by minyoung0503** [comment]
### 완료한일
* 리뷰 박스 컴포넌트 구현

### 해야할 일
* 코드 정리하고 PR 올리기

### 어려운 일
